compiler/test/dotty/tools/dotc/semanticdb/DiffAssertions.scala

```diff
@@ -0,0 +1,34 @@
+package dotty.tools.dotc.semanticdb
+
+import scala.collection.JavaConverters._
+
+object DiffAssertions {
```

Comment:
We should use `dotty.tools.dotc.util.DiffUtil` for comparison the same way we do for other tests – look it up in the ParallelTesting.scala file for the example of usage.

Reply:
I didn't need diff tools in the end